main.py

```diff
@@ -41,12 +41,23 @@ def get_statuses():
     return data_manager.get_statuses()
 
 
-@app.route('/add-card', methods='POST')
+@app.route('/add-card', methods=['POST'])
 def add_card():
     next_board_num = data_manager.get_next_board_name()
     data_manager.add_new_board('board' + next_board_num)
 
 
+@app.route("/change-board-title", methods=['PUT'])
+@json_response
+def change_board_title():
+    req = request.get_json()
+    print(req)
+    new_title = req["newTitle"]
+    board_id = req["boardId"]
+    data_manager.rename_board(board_id, new_title)
+
```

Comment:
When you have nothing to put in the response body, it is recommended to use status code 204, and leave the response body empty. 
(Now we have 'null' as a string in response body, you can see it on the picture attached.)
(
![2019-09-18_15h18_32](https://user-images.githubusercontent.com/10021819/65152283-f0cc2080-da27-11e9-9b4a-fa71c1f069fc.png)
Here is a guide how to do it in Flask: [https://www.erol.si/2018/03/flask-return-204-no-content-response/](https://www.erol.si/2018/03/flask-return-204-no-content-response/)